Enable system installation of OpenAL and Jemalloc by default since the
bundled versions don't play nicely with musl libc.

--- a/launcher/Application.cpp
+++ b/launcher/Application.cpp
@@ -639,12 +639,12 @@ Application::Application(int& argc, char** argv) : QApplication(argc, argv)
         m_settings->registerSetting("OnlineFixes", false);
 
         // Native library workarounds
-        m_settings->registerSetting("UseNativeOpenAL", false);
-        m_settings->registerSetting("CustomOpenALPath", "");
+        m_settings->registerSetting("UseNativeOpenAL", true);
+        m_settings->registerSetting("CustomOpenALPath", "/usr/lib/libopenal.so.1");
         m_settings->registerSetting("UseNativeGLFW", false);
         m_settings->registerSetting("CustomGLFWPath", "");
-        m_settings->registerSetting("UseNativeJemalloc", false);
-        m_settings->registerSetting("CustomJemallocPath", "");
+        m_settings->registerSetting("UseNativeJemalloc", true);
+        m_settings->registerSetting("CustomJemallocPath", "/usr/lib/libjemalloc.so.2");
 
         // Performance related options
         m_settings->registerSetting("EnableFeralGamemode", false);
